docs/topics/multiplatform/multiplatform-share-on-platforms.md

```diff
@@ -268,18 +262,12 @@ it and call `runBlocking` from a source set that is shared between the JVM and n
 
 ### Use native libraries in the hierarchical structure
 
-You can use platform-dependent libraries like Foundation, UIKit, and POSIX in source sets shared among several native 
+You can use platform-dependent libraries like `Foundation`, `UIKit`, and `posix` in source sets shared among several native 
```

Comment:
NIT:
I believe `POSIX` should be still written with capital letters. That is their official style.